packages/react/src/components/ContextualMenu/ContextualMenuItemWrapper/ContextualMenuSplitButton.tsx

```diff
@@ -183,6 +183,7 @@ export class ContextualMenuSplitButton extends ContextualMenuItemWrapper {
       checked: item.checked,
       iconProps: item.iconProps,
       id: this._dismissLabelId,
+      onClick: item.onClick,
```

Comment:
Would love to see a test since tests are part of the way you investigated the existing behavior :)

Reply:
done!

Reply:
Awesome! Added a comment but approving.